isis/src/base/apps/caminfo/caminfo.cpp

```diff
@@ -286,6 +286,67 @@ namespace Isis{
           camstats->append(MakePair("ObliqueResolutionMaximum", cg["ObliqueResolutionMaximum"][0]));
         }
 
+        // Extracts camstat data from existing CameraStatistics Table in cube label
+        if(ui.GetBoolean("USECAMSTATSTBL")) {
```

Comment:
We generally don't want to both run camstats and use the camstats label in this. You should add a check before the "CAMSTATS" block checks the user didn't enable both. The exclude in the XML only works for GUI, not the command line.

Reply:
I assumed it was handled in the command line as well since when trying to set both true, I get the error `**USER ERROR** Parameter [USECAMSTATSTBL] must NOT be used if parameter [CAMSTATS] equates to true.`

Reply:
I must have that incorrect then. You can ignore this.

Reply:
This behavior does not address the main motivation for adding this option. 

CAMSTATS and USECAMSTATSLBL are not mutually exclusive. 

Only consider USECAMSTATSLBL when the user would prefer to use the camstats table pre-calculated with specific parameters earlier in processing for efficiency and consistency reasons. This condition is achieved when CAMSTATS=TRUE and USECAMSTATSLBL=TRUE.

camstats is typically ran as part of standard base processing since it is needed by qmos and other apps.

The other use case is when users would rather specify their own values (LINC/SINC) regardless if a camstats run already exists in the table. In this case CAMSTATS=TRUE and USECAMSTATSLBL=FALSE (consistent with current behavior) and LINC/SINC are then used in this case.

USECAMSTATSLBL specifies a preference to use an existing table if it exists but if it doesn't, go ahead and calculate it with the parameters the user provides. An error thrown because the table doesn't exist would be very unfortunate/irritating. No error should occur if the table does not exist for any condition with this approach.

This change expedites efficiency and flexibility for camera statistics.